staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources.go

```diff
@@ -235,6 +238,15 @@ func (o *APIResourceOptions) RunAPIResources() error {
 				strings.Join(r.APIResource.Categories, ",")); err != nil {
 				errs = append(errs, err)
 			}
+		case "json":
```

Comment:
Could you please check how other commands handle json printing(i.e. commands support `--output`). We need to also support yaml and that's why, we need to use JsonYamlPrinter.

Since this version of this patch looks hacky;

/hold

Reply:
@XDRAGON2002  It is worth to check how `wait` command handles this https://github.com/kubernetes/kubernetes/blob/369534c6ec625ed9b251b453215c9578cdeb59bc/staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go#L152